src/parser.rs

```diff
@@ -599,18 +598,19 @@ impl Library {
                                     "name" => field_name = attr.value.clone(),
                                     _ => {},
                                 }
+                                break;
```

Comment:
Outside the `match` not the loop. The match block only runs for the current iteration of the loop, not trying to match all elements until exhausted.